leasing/serializers/rent.py

```diff
@@ -564,7 +578,63 @@ def validate(self, data):
         validate_seasonal_day_for_month(start_day, start_month)
         validate_seasonal_day_for_month(end_day, end_month)
 
-        return data
+    def validate_override_receivable_type_value(self, rent_data: dict) -> None:
+        """
+        Override receivabletype is mandatory for AKV and KuVa leases, and not
+        used by MaKe/Tontit.
+
+        It is only used in index rents, fixed rents, and manual rents, because
+        these rent types can generate automatic invoices.
+
+        Raises: serializers.ValidationError
+        """
+        override_receivable_type = rent_data.get("override_receivable_type")
+
+        if not override_receivable_type:
+            # Empty override receivabletype input should be allowed, because it
+            # is always empty for all MaKe rents, and those must be allowed.
+            return
+
+        elif not is_akv_or_kuva_service_unit_id(
+            override_receivable_type.service_unit_id
+        ):
+            # All MaKe receivabletypes should be rejected regardless of rent
+            # type, because MaKe doesn't use the override feature, and if any
+            # other service unit would use MaKe's receivabletypes, it would be
+            # a mistake.
+            raise serializers.ValidationError(
+                _(
+                    f'Override receivabletype "{override_receivable_type.name}" was unexpected. '
+                    "Override receivabletype is not used by MaKe/Tontit."
+                )
+            )
+
+        elif is_akv_or_kuva_service_unit_id(override_receivable_type.service_unit_id):
```

Comment:
Is this an else of the previous elif (which would then be an if)?

Reply:
No, it's part of the single top-level branching. This logic will probably change now that I'll add use_override_receivable_type to service units.

Reply:
... but because first if can do early return, there is no need for `elif` in the second branch

Reply:
This was overhauled in a later commit